modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesConstants.java

```diff
@@ -35,4 +35,9 @@
      */
     public static final String TENANT_SEPARATOR = "/";
 
+    /**
+     * The prefix which distinguishes a durable subscription.
+     */
+    public static final String DURABLE_SUBSCRIPTION_QUEUE_PREFIX = "carbon:";
+
```

Comment:
The value 'carbon' does not suggest any durable property. Can we rename this to something like 'carbon:durable"? This will be useful when viewing these raw data from the database for debugging.